README.md

```diff
@@ -15,8 +16,7 @@
 - [Website](https://tinacms.org/)
   - [Blog](https://tinacms.org/docs/)
   - [Docs](https://tinacms.org/docs/)
-- [Community](https://tinacms.org/docs/)\
-  - [Forum](https://community.tinacms.org/)
+- [Community](https://community.tinacms.org/)
```

Comment:
Should we just call this Forum?

Reply:
I updated it to be "Community Forum" not sure if that is better or if just "Forum" would be better